include/librealsense2/dds/topics/notifications/notifications-msg.h

```diff
@@ -0,0 +1,118 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#pragma once
+#include <string>
+#include "librealsense2/h/rs_sensor.h"
+#include "notificationsPubSubTypes.h"
+
+namespace librealsense {
+namespace dds {
+namespace topics {
+namespace device {
+class notifications
+{
+public:
+
+    using type = raw::device::notificationsPubSubType;
+
+#pragma pack( push, 1 )
+    enum class msg_type : uint16_t
+    {
+        DEVICE_HEADER,
+        STREAM_HEADER,
+        VIDEO_STREAM_PROFILES,
+        MOTION_STREAM_PROFILES,
+        MAX_MSG_TYPE
+    };
+
+    struct device_header
+    {
+        size_t num_of_streams;
+    };
+
+    struct stream_header
+    {
+        uint8_t index;           // Index of the current stream [0-(num_of_streams-1)]
+        char name[32];           // Stream name
+        size_t num_of_profiles;  // Profile count for this stream
+    };
+
+
+    struct video_stream_profile
+    {
+        int8_t index;           // Sensor index (Normally )
+        int16_t uid;            // Stream unique ID
+        int8_t framerate;
+        rs2_format format;      // Transfer as uint8_t?
+        rs2_stream type;        // Transfer as uint8_t?
+        int16_t width;          
+        int16_t height;         
+        //intrinsics - TODO
+    };
+
+    struct motion_stream_profile
+    {
+        int8_t index;           // Sensor index (Normally )
+        int16_t uid;            // Stream unique ID
+        int8_t framerate;
+        rs2_format format;      // Transfer as uint8_t?
+        rs2_stream type;        // Transfer as uint8_t?
+    };
+
+    struct video_stream_profiles
+    {
+        std::vector<video_stream_profile> profiles_vec;
+    };
+
+    struct motion_stream_profiles
+    {
+        std::vector<motion_stream_profile> profiles_vec;
+    };
+
+#pragma pack( pop )
+
+    static std::string construct_notifications_topic_name( const std::string& topic_root )
```

Comment:
Any reason it's not a shorter `construct_topic_name`?

Reply:
No, will be shorten.

Reply:
Done